js/package.json

```diff
@@ -24,17 +24,18 @@
   },
   "homepage": "https://github.com/borgbackup/borgweb#readme",
   "devDependencies": {
-    "babelify": "^6.1.2",
-    "browser-sync": "^2.7.13",
-    "browserify": "^10.2.4",
+    "@babel/core": "^7.0.0-beta.51",
```

Comment:
this is a new dependency and it is beta. is listing it necessary and could we use something non-beta?

Reply:
babel 7 is in just beta but it works fine if you go to V6 some error rise I used V7 beta for a while now it works fine and also has not any security issue
good thing is package json  had more vulnerability and also high level ones than just jquery now 2 low ones left :)